Monika After Story/game/script-farewells.rpy

```diff
@@ -947,6 +947,25 @@ label bye_going_somewhere:
         $ _history_list.pop()
         m 1lksdlc "..."
         m 1eksdlb "What am I saying? Of course I'll go with you!"
+        if mas_isMoniLove(higher=True):
+        m 1hub "Oh, okay!"
+        menu:
+            m 3tub "Where are you taking me?"
+            "School.":
+                m 1eka "Ahaha [Player], I cant wait to see all your friends!"
+            "Work.":
+                m 1eka "How Fun!"
+            "A date.":
+                m 1eka "How Romantic [Player]!"
+            "Vacation.":
+                m 1eka "Im sure its going to be amazing"
```

Comment:
Missing apostrophe in "Im".

Missing apostrophe in "its".

Missing a period.